javascript-prep/src/components/DetailPage/DetailPage.js

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
 import ImportantLinksComponent from './ImportantLinksComponent';
-
+import QuestionAnswerComponent from './QuestionAnswerComponent';
+import '../../css/detail.css';
 export default function DetailPage(props) {
     let json = require(`../../staticData/${props.match.params.id.substr(1)}.json`)
     return (
-        <div>
-            <h1>{json.topic}</h1>
+        <div className="detail-container" >
+            <div className="qa-container">
+                <h1 style={{textAlign: 'center'}}>{json.topic}</h1>
+                <QuestionAnswerComponent />
```

Comment:
pass on the question_answers array to the component

Reply:
done